app/api/connectors/httpparsers/MtdIdLookupHttpParser.scala

```diff
@@ -16,26 +16,20 @@
 
 package api.connectors.httpparsers
 
-import api.connectors.MtdIdLookupOutcome
-import api.models.errors.{InternalError, InvalidBearerTokenError, NinoFormatError}
-import play.api.http.Status.{FORBIDDEN, OK, UNAUTHORIZED}
+import api.connectors
+import api.connectors.MtdIdLookupConnector
+import play.api.http.Status.OK
 import play.api.libs.json._
 import uk.gov.hmrc.http.{HttpReads, HttpResponse}
 
 object MtdIdLookupHttpParser extends HttpParser {
 
   private val mtdIdJsonReads: Reads[String] = (__ \ "mtdbsa").read[String]
 
-  implicit val mtdIdLookupHttpReads: HttpReads[MtdIdLookupOutcome] = (_: String, _: String, response: HttpResponse) => {
+  implicit val mtdIdLookupHttpReads: HttpReads[connectors.MtdIdLookupConnector.Outcome] = (_: String, _: String, response: HttpResponse) => {
     response.status match {
-      case OK =>
-        response.validateJson[String](mtdIdJsonReads) match {
-          case Some(mtdId) => Right(mtdId)
-          case None        => Left(InternalError)
-        }
-      case FORBIDDEN    => Left(NinoFormatError)
-      case UNAUTHORIZED => Left(InvalidBearerTokenError)
-      case _            => Left(InternalError)
```

Comment:
Does this mean that now the old 500s get mapped differently?

Reply:
No. This code is essentially moved to the `MtdIdLookupService`. This allows the http parser & connector to not be concerned about the interpretation of errors coming from downstream. They just honestly report them to the service: 'this is what the lookup service returned'.